### ISSUE TITLE
Template for Hello World with Powertools doesn't exist for Python3.10

### ISSUE THREAD

**Issue by valerena** [aws/aws-sam-cli-app-templates#383]
### Description:

Template for Hello World with Powertools doesn't exist for Python3.10 even though it's one of the options.

The reference is [in the manifest file](https://github.com/aws/aws-sam-cli-app-templates/blob/master/manifest-v2.json#L1069), but there's [no actual directory](https://github.com/aws/aws-sam-cli-app-templates/tree/master/python3.10) (likely because Python 3.10 was just launched and the Powertools samples were added recently too)

### Steps to reproduce:
```
$ sam init
1
7
11
```
(full output ->)
```
$ sam init

You can preselect a particular runtime or package type when using the `sam init` experience.
Call `sam init --help` to learn more.

Which template source would you like to use?
        1 - AWS Quick Start Templates
        2 - Custom Template Location
Choice: 1

Choose an AWS Quick Start application template
        1 - Hello World Example
...
        7 - Hello World Example With Powertools
...
Template: 7

Which runtime would you like to use?
        1 - dotnet6
...
        11 - python3.10
Runtime: 11
```

### Observed result:
```
Error: Please verify your location. The following types of location are supported:

* Github: gh:user/repo (or) https://github.com/user/repo (or) git@github.com:user/repo.git
          For Git repositories, you must use location of the root of the repository.

* Mercurial: hg+ssh://hg@bitbucket.org/repo

* Http(s): https://example.com/code.zip

* Local Path: /path/to/code.zip
```

### Expected result:
*success*

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

Likely happening everywhere

1. OS: 
2. If using SAM CLI, `sam --version`:
3. AWS region:

`Add --debug flag to any SAM CLI commands you are running`

**Comment by mildaniel** [comment]
Thanks @valerena for bringing this up. Looks like we might need to improve our process here around introducing new runtimes and ensuring that the runtimes in the manifest match up with the ones in the repo.

**Comment by qingchm** [comment]
This has been addressed in our most recent release 1.79.0 (https://github.com/aws/aws-sam-cli/releases/tag/v1.79.0). Closing the issue here. Thanks for the ticket!
